lora-phy/Cargo.toml

```diff
@@ -14,17 +14,17 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
-defmt = { version = "0.3" }
+defmt = { version = "0.3", optional = true }
 lora-modulation = { path = "../lora-modulation", version = ">=0.1.2" }
-lorawan-device = { path = "../lorawan-device", default-features = false, version = "0.12", features = [
-    "defmt",
-], optional = true }
+lorawan-device = { path = "../lorawan-device", default-features = false, version = "0.12", optional = true }
 num-traits = { version = "0.2", default-features = false }
 embedded-hal = { version = "1" }
 embedded-hal-async = { version = "1" }
 document-features = "0.2.8"
 
 [features]
+
+defmt = ["dep:defmt", "lorawan-device/defmt", "lora-modulation/defmt"]
```

Comment:
Please also add description for the feature (eventually will improve API docs once we pull in [document-features](https://crates.io/crates/document-features) like we do for lorawan-device):
`##  Enable defmt logging framework`